apps/elf-council-frontend/src/ui/proposals/GSCProposalsSection.tsx

```diff
@@ -60,40 +59,39 @@ export default function GSCProposalsSection({
   const isTailwindSmallScreen = useIsTailwindSmallScreen();
   const isTailwindLargeScreen = useIsTailwindLargeScreen();
 
-  const unverifiedProposals = useGSCUnverifiedProposals(
-    proposalsJson.proposals,
-  );
-  const allProposals = proposalsJson.proposals.concat(unverifiedProposals);
-
   const activeProposals = useFilteredProposals(
     TabId.ACTIVE,
-    allProposals,
+    proposalsJson.proposals,
     currentBlockNumber,
   );
   const pastProposals = useFilteredProposals(
     TabId.PAST,
-    allProposals,
+    proposalsJson.proposals,
     currentBlockNumber,
   );
 
   const setDefaultActiveProposal = useCallback(() => {
     setSelectedProposalId(activeProposals?.[0]?.proposalId);
     setSelectedProposal(activeProposals?.[0]);
-  }, [activeProposals]);
+    isTailwindLargeScreen && activeProposals.length && setIsModalOpen(true);
```

Comment:
nit: this shorthand feels a bit too clever, can you refactor this into an if-statement instead?